task5/src/main/java/com/clouway/adapter/persistence/PersistentSessionRepository.java

```diff
@@ -133,4 +133,21 @@ public void refreshSessionTime(Session session) {
       }
     }
   }
+
+  public int getActiveSessions() {
```

Comment:
In the `Login page` you have 
```java
Integer usersOnline = sessionRepository.getActiveSessions();
```
I mean the result from method what returns primitive type is teken from variable of non primitive type. Good proggrammer would not do it this way.